tratto/src/main/java/star/tratto/util/javaparser/JavaParserUtils.java

```diff
@@ -0,0 +1,1078 @@
+    }
+
+    /**
+     * Returns the String variable declaration represented by a JavaParser
+     * VariableDeclarator. Recall that a JavaParser field declaration may
+     * declare multiple variables. Each variable is separated into individual
+     * declarations.
+     *
+     * @param field the JavaParser field declaration
+     * @param variable the JavaParser variable declaration
+     * @return a string representation of the declaration of {@code variable}.
+     * The declaration follows the format:
+     *  "(modifiers) (type) (name)( = initial value);"
+     */
+    public static String getVariableDeclaration(FieldDeclaration field, VariableDeclarator variable) {
+        return (String.join("", field.getModifiers().stream().map(Node::toString).toList()) +
+                variable.getTypeAsString() + " " +
+                variable.getNameAsString() +
+                (variable.getInitializer().isPresent()
+                        ? " = " + variable.getInitializer().get()
+                        : "") +
+                ";").trim();
+    }
+
+    /**
+     * Returns the String field declaration represented by a JavaParser
+     * {@link ResolvedFieldDeclaration} as it would appear in source code.
+     *
+     * @param resolvedField a field declaration
+     * @return a string representation of the declaration of
+     * {@code resolvedField}. The declaration follows the format:
+     *  "(access-specifier) (static) (type) (name);"
+     */
+    public static String getFieldDeclaration(
+            ResolvedFieldDeclaration resolvedField
+    ) {
+        boolean hasAccessSpecifier = !resolvedField.accessSpecifier().asString().equals("");
+        boolean isStatic = resolvedField.isStatic();
+        return ((hasAccessSpecifier ? resolvedField.accessSpecifier().asString() + " " : "") +
+                (isStatic ? "static " : "") +
+                getTypeWithoutPackages(resolvedField.getType().describe()) + " " +
+                resolvedField.getName() +
+                ";");
+    }
+
+    /**
+     * Gets the String field declaration represented by a JavaParser
+     * {@link ResolvedFieldDeclaration}.
+     *
+     * @param resolvedField a field declaration
+     * @param modifier an integer representing the field modifiers
+     * @return a string representation of the declaration. The declaration
+     * follows the format:
+     *  "(modifiers) (type) (name);"
+     */
+    public static String getFieldDeclaration(
+            ResolvedFieldDeclaration resolvedField,
+            int modifier
+    ) {
+        return ((modifier == 0) ? "" : (java.lang.reflect.Modifier.toString(modifier) + " ") +
+                getTypeWithoutPackages(resolvedField.getType()) + " " +
+                resolvedField.getName() +
+                ";").trim();
+    }
+
+    /**
+     * Returns the signature of a JavaParser callable declaration. Uses the
+     * method source code and removes method body, contained comments, the
```

Comment:
I don't understand this method's implementation strategy.  It takes the full source code and deletes text until it gets the signature.  Please document (here and in the below method) why it wouldn't work to start from nothing and build up the signature, as the getFieldDeclaration methods above do.